packages/office-ui-fabric-react/src/common/DocPage.types.ts

```diff
@@ -135,4 +135,39 @@ export interface IDocPageProps {
     title: string;
     section: JSX.Element;
   }[];
+
+  jsonDocs?: IPageJson;
+}
+
+export interface ITokenJson {
+  text: string;
+  hyperlinkedPage?: string;
+}
+
+export interface ITableRowJson {
+  name: string;
+  typeTokens: ITokenJson[];
+  defaultValue?: string;
+  descriptionHtml: string;
+  deprecated: boolean;
+  deprecatedMessage?: string;
+}
+
+export interface IEnumTableRowJson {
```

Comment:
Just wondering if the `Json` ending is really bringing a lot more clarity to what this interface is representing? Will the semantics be worse if we drop the Json part from the naming?